packages/@dcl/sdk-commands/src/commands/start/server/endpoints.ts

```diff
@@ -137,6 +137,26 @@ export async function setupEcs6Endpoints(
     return res
   })
 
+  router.get('/explorer/:address/wearables', async (ctx) => {
+    const u = new URL(ctx.url.toString())
+    u.host = catalystUrl.host
+    u.protocol = catalystUrl.protocol
+    u.port = catalystUrl.port
+    const req = await fetch(u.toString(), {
+      headers: {
+        connection: 'close'
+      },
+      method: 'GET'
```

Comment:
taken from lambdas endpoint 🙏 
```suggestion
      method: ctx.request.method,
      body: ctx.request.method === 'get' ? undefined : ctx.request.body
```